target/linux/realtek/dts/rtl8393_hpe_1920-48g-poe.dts

```diff
@@ -16,6 +16,25 @@
 		alarm-gpios = <&gpio1 6 GPIO_ACTIVE_LOW>;
 		#cooling-cells = <2>;
 	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		mode_button {
+			label = "mode";
+			linux,code = <BTN_0>;
+			gpios = <&gpio0 23 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		mode_led {
+			label = "green:mode";
+			gpios = <&gpio1 8 GPIO_ACTIVE_HIGH>;
```

Comment:
You should preferably use the "function" and "color" properties instead of label.
See https://mjmwired.net/kernel/Documentation/devicetree/bindings/leds/common.yaml

In absence of an appropriate LED_FUNCTION_* macro, you can just use a string (e.g `function = "mode"`)

See `default-state` and `linux,default-trigger` to set the default state/trigger instead of using the 01_leds script.